.github/workflows/buildwebsite.yml

```diff
@@ -0,0 +1,38 @@
+name: continuous build - docs
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  unittest-gpu:
+    runs-on: ubuntu-latest
+    strategy: 
+      fail-fast: false
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install Linux dependencies
+        run: sudo apt-get install libopenblas-dev
+
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+          architecture: x64
+
+      - name: Install Other Dependencies
+        run: |
+          python -m pip install --user --quiet --upgrade pip
+          python3 -m pip install setuptools pytest pytest-cov contextvars
```

Comment:
contextvars is not needed on python 3.7. Why do we need pytest for the website build?